multicluster/apis/multicluster/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
why is this needed?

Reply:
let me check, I think it's introduced by golang 1.17.

Reply:
this line is nothing to do with setup doc, I run `make` once and it's added automatically, it's new from golang 1.17 https://golang.org/doc/go1.17, I added it here considering it's only one line change.